L_ZWay2.lua

```diff
@@ -741,7 +747,7 @@ function _G.updateChildren (d)
         local id = vtype .. altid
         setVar (id, inst.metrics.level, SID.ZWay, zDevNo)
         setVar (id .. "_LastUpdate", inst.updateTime, SID.ZWay, zDevNo)
-
+        setVar ("CommFailure", inst.metrics.isFailed, SID.HaDevice, zDevNo)
```

Comment:
Not quite that simple.  There's also a `CommFailureTime` variable.  This is generally managed through the luup.set_failure() call documented [here on the MCV Wiki](http://wiki.micasaverde.com/index.php/Luup_Lua_extensions#function:_set_failure), but I need to check the detail of the openLuup implementation.